stdlib/public/core/Span/Span.swift

```diff
@@ -12,236 +12,270 @@
 
 // A Span<Element> represents a span of memory which
 // contains initialized instances of `Element`.
+@_disallowFeatureSuppression(NonescapableTypes)
 @frozen
-public struct Span<Element: ~Copyable & ~Escapable>: Copyable, ~Escapable {
+public struct Span<Element: ~Copyable & ~Escapable>
+: ~Escapable, Copyable, BitwiseCopyable {
   @usableFromInline let _pointer: UnsafeRawPointer?
 
   @usableFromInline @inline(__always)
   var _start: UnsafeRawPointer { _pointer.unsafelyUnwrapped }
 
   @usableFromInline let _count: Int
 
-  @_alwaysEmitIntoClient
-  internal init(
-    _unchecked start: UnsafeRawPointer?,
+  @_disallowFeatureSuppression(NonescapableTypes)
+  @usableFromInline @inline(__always)
+  @lifetime(immortal)
```

Comment:
Should be `@lifetime(pointer)`. The resulting Span depends on the validity of the pointer.

Reply:
We should be able to use `@lifetime(borrow pointer)` after https://github.com/swiftlang/swift-syntax/pull/2894 is merged.